BuyAndSellStock/README.md

```diff
@@ -0,0 +1,126 @@
+# Buy And Sell Stock
+
+### https://leetcode.com/problems/best-time-to-buy-and-sell-stock/description/
+
+## First Solution
```

Comment:
These solutions aren't substantially different, especially when you consider the Big-O complexity. It would be good to work through different solutions of varying time and space complexities (even if they're not the most optimal). 

Also, experiment with the Java Stream API and see if you can produce a working solution with just a single stream of chained functions.

Reply:
Ill check this out. Will update ThirdSolution with something that implements the Java Stream API